netdata.spec.in

```diff
@@ -239,11 +239,14 @@ export CFLAGS="${CFLAGS} -fPIC" && ${RPM_BUILD_DIR}/%{name}-%{version}/packaging
 %if 0%{?centos_ver:1}
 %if %{centos_ver} < 8
 export CFLAGS="${CFLAGS} -fPIC" && ${RPM_BUILD_DIR}/%{name}-%{version}/packaging/bundle-libbpf.sh ${RPM_BUILD_DIR}/%{name}-%{version} centos7
+export CFLAGS="${CFLAGS} -fPIC" && ${RPM_BUILD_DIR}/%{name}-%{version}/packaging/bundle-ebpf-co-re.sh ${RPM_BUILD_DIR}/%{name}-%{version} centos7
```

Comment:
In this and the next two cases, you can drop the distro name at the end of the command, as the script you’re invoking doesn’t actually use it.

Reply:
I fixed with last commit, thanks!